{"name": "Se observan los campos UserName y Password -- @1.1 ", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"C:\\ProgramFiles\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1461, in run\n    match.run(runner.context)\n  File \"C:\\ProgramFiles\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1908, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\01Login.py\", line 11, in step_impl\n    context.driver.close()\n    ^^^^^^^^^^^^^^\n  File \"C:\\ProgramFiles\\Python311\\Lib\\site-packages\\behave\\runner.py\", line 214, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Given Iniciando Navegador en la Web https://demo.opencart.com/admin", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"C:\\ProgramFiles\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1461, in run\n    match.run(runner.context)\n  File \"C:\\ProgramFiles\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1908, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\01Login.py\", line 11, in step_impl\n    context.driver.close()\n    ^^^^^^^^^^^^^^\n  File \"C:\\ProgramFiles\\Python311\\Lib\\site-packages\\behave\\runner.py\", line 214, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1681583666221, "stop": 1681583666223}, {"name": "When Escribiendo campo  Username :\"demo\"", "status": "skipped", "start": 1681583666224, "stop": 1681583666224}, {"name": "When Escribiendo campo password :\"125635\"", "status": "skipped", "start": 1681583666224, "stop": 1681583666224}, {"name": "When Click en boton Login", "status": "skipped", "start": 1681583666224, "stop": 1681583666224}, {"name": "Then validar Notificacion  \"No match for Username and/or Password.\"", "status": "skipped", "start": 1681583666224, "stop": 1681583666224}], "parameters": [{"name": "Username", "value": "demo"}, {"name": "Password", "value": "125635"}], "start": 1681583666219, "stop": 1681583666224, "uuid": "59996a67-d4fa-4d8c-9886-a79f268f0ecd", "historyId": "81fc9d70eaf221506fb15211ff3f2fa9", "fullName": "Inicio de Sesion con Datos Invalidos: Se observan los campos UserName y Password", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Inicio de Sesion con Datos Invalidos"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}